JakeUni95/interview-questions-frontend
# Replaces Skill ID with its name in the URL

The URL now shows the selected skills name(in slug form) rather than its ID

## How to test
Run CMS locally.
Make sure that frontend files are built by running npm run build.
Run frontend locally.
Click through pages and verify visual changes to the URL.
